src/ai/backend/manager/models/association_container_registries_groups.py

```diff
@@ -0,0 +1,31 @@
+from __future__ import annotations
+
+import logging
+from typing import Sequence
+
+import sqlalchemy as sa
+
+from ai.backend.logging import BraceStyleAdapter
+
+from .base import GUID, Base, IDColumn
+
+log = BraceStyleAdapter(logging.getLogger(__spec__.name))  # type: ignore
+
+__all__: Sequence[str] = ("AssociationContainerRegistriesGroupsRow",)
+
+
+class AssociationContainerRegistriesGroupsRow(Base):
+    __tablename__ = "association_container_registries_groups"
+    id = IDColumn()
+    registry_id = sa.Column(
+        "registry_id",
+        GUID,
+        sa.ForeignKey("container_registries.id", onupdate="CASCADE", ondelete="CASCADE"),
```

Comment:
We recommend not to use foreign key constraint anymore in backend.ai.
Please check [this code](https://github.com/lablup/backend.ai/blob/main/src/ai/backend/manager/models/vfolder.py#L455-L459) how to load related rows without a foreign key constraint.